lib/bolt/cli.rb

```diff
@@ -443,13 +475,21 @@ def execute(options)
       message = 'There may be processes left executing on some nodes.'
 
       if options[:mode] == 'plan'
-        executor = Bolt::Executor.new(@config, options[:noop], true)
+        unless options[:nodes].empty?
+          if options[:task_options]['nodes']
+            raise Bolt::CLIError,
+                  "A plan's 'nodes' parameter may be specified using the --nodes option, but in that " \
+                  "case it must not be specified as a separate nodes=<value> parameter."
```

Comment:
I have updated the message as requested, but I'm unsure whether we should somehow indicate that the 'nodes'  must not be specified in the JSON passed to the `--params` option either when already specified using the `--nodes` options.